src/transformers/utils/doc.py

```diff
@@ -1140,6 +1142,15 @@ def docstring_decorator(fn):
         )
         if real_checkpoint is not None:
             code_sample = FAKE_MODEL_DISCLAIMER + code_sample
+        if revision is not None:
+            if "refs/pr/" not in revision:
+                raise ValueError(
+                    f"The provided revision '{revision}' is incorrect. It should point to"
+                    " a pull request reference on the hub like 'refs/pr/6'"
+                )
```

Comment:
personal pref but perhaps use a more strict regex here rather than just checking if the substring is present?

Reply:
Yep I'll do that 😉